style.css

```diff
@@ -433,6 +433,13 @@ input[type="email"] {
   line-height: 2;
 }
 
+input[type="email"][disabled],
```

Comment:
May be good to rely on `:disabled` instead of `[disabled]` but I am open to suggestion https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled

Reply:
I went for `[disabled]` to be consistent with what was above. But I do prefer `:disabled` as well, so will change this